content/media_content.php

```diff
@@ -5,6 +5,18 @@
 				</div>
 			</div>
 			<div class="post">
+				<h2 class="title"><a href="#">2016 Unveil Video:</a></h2>
+				<div class="entry">
+					<iframe width="560" height="315" src="https://www.youtube.com/embed/OIRAUX4TTXc" frameborder="0" allowfullscreen></iframe>
+				</div>
+			</div>
+			<div class="post">
+				<h2 class="title"><a href="#">2015 Unveil Video:</a></h2>
```

Comment:
Why the link in the `<h2>`?
Kinda seems like a 👃 to me...I would not recommend using this as a solution to a back-to-top link. A back-to-top link is bad UX anyway, but clicking a title is not simantically clear. A better use you be to do something like:
```html
<h2 id="2015Unveil">
  <a href="#2015Unvail">2015 Unveil Video:</a>
</h2>
```

Now when they click the title, if it can, it will scroll that heading to the top.
Also it will update the location, so now you can link directly to the specific section in emails and what not. That would be the expected behavior of these headings to a typical user.

If you would like to keep that, then I would use a `<span class="post__subheading"><a href="#">Back to Top</a></span>` inside of your `<h2>` and position it to the right.

But again a Back to the Top link is generally considered bad UX.